examples/seznam.py

```diff
@@ -0,0 +1,39 @@
+"""Seznam Login Example"""
+
+import os
+import uvicorn
+from fastapi import FastAPI
+from fastapi import Request
+
+from fastapi_sso.sso.seznam import SeznamSSO
+
+CLIENT_ID = os.environ["CLIENT_ID"]
+CLIENT_SECRET = os.environ["CLIENT_SECRET"]
+
+app = FastAPI()
+
+sso = SeznamSSO(
+    client_id=CLIENT_ID,
+    client_secret=CLIENT_SECRET,
+    redirect_uri="http://localhost:5000/auth/callback",
+    allow_insecure_http=True,
+)
+
+
+@app.get("/auth/login")
+async def auth_init():
+    """Initialize auth and redirect"""
+    with sso:
+        return await sso.get_login_redirect()
+
+
+@app.get("/auth/callback")
+async def auth_callback(request: Request):
+    """Verify login"""
+    with sso:
+        user = await sso.verify_and_process(request, params={"client_secret": CLIENT_SECRET})
```

Comment:
Is passing the client secret explicitly in `params` neccessary? I would guess oauthlib adds the client secret to the request uri when doing `prepare_token_request` in `verify_login` method of `SSOBase` :thinking: If this is specific to Seznam, I'd probably also add it to documentation, or maybe emphasize it in the comment here so that users who come to documented examples actually notice that this is some specialty :)

Reply:
Yes, unfortunately it is needed. It took me a while to figure it out, but it's also described in the documentation - "2. Převod kódu na token" contains a "client_secret" field. Without it, Seznam API returns the response "{'error': 'invalid_client'}".

I added a comment to the example
https://github.com/tomasvotava/fastapi-sso/pull/194/commits/3be5d2e72b5ca08765593161e9663675c32be405